src/content/docs/troubleshooting.mdx

```diff
@@ -265,6 +265,26 @@ installation. This can be fixed by running any of:
 1. `flutter run` in your project's root directory and targeting an iOS
    simulator/device.
 
+## Unsupported class file major version 65
+
+Sometimes when running `shorebird` commands that involve gradle processes you might face
+this error: `Unsupported class file major version 65`, (or 66, the number varies sometimes).
+
+This error usually happens because gradle is being executed with an incompatible Java version.
+
+You can check the specific Gradle version that your app is using by running
+`shorebird doctor -v` inside the project folder. This command will also print the java version that
+shorebird is using.
+
+Refer to the [Gradle's official documentation](https://docs.gradle.org/current/userguide/compatibility.html)
+to check which Java version works with which Gradle release.
+
+The most safe way to set a specific java installation for shorebird to use, is to set the `JAVA_HOME`
+environment variable.
```

Comment:
How does one find the right `JAVA_HOME` directory?

Reply:
Based on the the compatibility matrix docs that I linked above from gradle.

Since the user can install java in many different ways or places, I am unsure how explain it better. I am open to suggestions though

Reply:
Right, the docs should say this :)

Reply:
Trying tweaking the phrasing a little bit more, let me know if this is more on what you were thinking.

Bear with me on these docs PR, I tend get wordy and this gets worse on english since it is not my main language 😅